src/ashwini/ExamPro.java

```diff
@@ -0,0 +1,42 @@
+package ashwini;
+
+public class ExamPro {
+	// to find all pairs whose sum is 6
+
+	void sumPairs(int[] b) {
+		for (int i = 0; i < b.length; i++) {
+			for (int j = i + 1; j < b.length; j++) {
+				if (b[i] + b[j] == 6) {
+					System.out.println(b[i] + " " + b[j]);
+					// System.out.println(b[j]);
+
+				}
+			}
+		}
+	}
+
+	// To find single occcurrence
+	void singleOcuurrence(int[] f) {
```

Comment:
the wrong output will be generated!

did you check n run your code? @ashwinipolpatil